integration-tests/deployment/ccip/state.go

```diff
@@ -290,6 +300,23 @@ func LoadChainState(chain deployment.Chain, addresses map[string]deployment.Type
 				return state, err
 			}
 			state.Receiver = mr
+		case deployment.NewTypeAndVersion(PriceFeed, deployment.Version1_0_0).String():
+			feed, err := aggregator_v3_interface.NewAggregatorV3Interface(common.HexToAddress(address), chain.Client)
+			if err != nil {
+				return state, err
+			}
+			if state.Feeds == nil {
+				state.Feeds = make(map[TokenDescriptor]*aggregator_v3_interface.AggregatorV3Interface)
+			}
+			desc, err := feed.Description(&bind.CallOpts{})
```

Comment:
Ah I see, I just realized how wacky the mock aggregator is. I would expect description to be something like "LINK/USD" (perhaps injected via constructor or set via a setter) but instead it seems like its "v0.8/..." which is... interesting.

Maybe a stretch goal to have a more sane mock but I guess that explains this code

Reply:
Yeah! It took me some time to get around this without changing the Mock TBH 😅. Maybe in next PR when I want to add more tokens I can change the aggregator constructor to take a description with a default one as well (don't know how this is done in Solidity yet).